crengine/src/private/lvfreetypeface.cpp

```diff
@@ -1120,11 +1136,11 @@ bool LVFreeTypeFace::getGlyphInfo(lUInt32 code, LVFont::glyph_info_t *glyph, lCh
     glyph->blackBoxY = (lUInt16)( FONT_METRIC_TO_PX( _slot->metrics.height ) );
     glyph->originX =   (lInt16)( FONT_METRIC_TO_PX( _slot->metrics.horiBearingX ) );
     glyph->originY =   (lInt16)( FONT_METRIC_TO_PX( _slot->metrics.horiBearingY ) );
-    glyph->width =     (lUInt16)( FONT_METRIC_TO_PX( myabs(_slot->metrics.horiAdvance )) );
+    glyph->width =     (lUInt16)( FONT_METRIC_TO_PX( myabs(_slot->metrics.horiAdvance + _synth_weight_advance_add )) );
     if (glyph->blackBoxX == 0) // If a glyph has no blackbox (a spacing
         glyph->rsb =   0;      // character), there is no bearing
     else
-        glyph->rsb =   (lInt16)(FONT_METRIC_TO_PX( (myabs(_slot->metrics.horiAdvance)
+        glyph->rsb =   (lInt16)(FONT_METRIC_TO_PX( (myabs(_slot->metrics.horiAdvance + _synth_weight_advance_add)
```

Comment:
You might be right:
I guess that when FreeType does `FT_Outline_Embolden()`, it may make it grow on the right - and I needed `FT_Outline_Translate()` to shift it a bit on the left, so the shape feels centered again in the original width/advance that I explicitely did not increase.

Now, you keep the added width from the embolding and make it part of width/advance.
I then guess origin_x does not need to change.
But then, `rsb` (the blank space between the glyph blackbox right side and the end of advance) too does not need to change.

Reply:
Ok I'll delete call FT_Outline_Translate(). But then I will have to delete/correct your comments again.  You won't be offended, will you? :)

Reply:
Of course I will, but I'll eventually get over it :)